sdk/kusto/mgmt-v2020_02_15/src/main/java/com/microsoft/azure/management/kusto/v2020_02_15/implementation/ClustersInner.java

```diff
@@ -1581,8 +1581,16 @@ public DiagnoseVirtualNetworkResultInner call(ServiceResponse<DiagnoseVirtualNet
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent
      * @return the List&lt;ClusterInner&gt; object if successful.
      */
-    public List<ClusterInner> list() {
-        return listWithServiceResponseAsync().toBlocking().single().body();
+    public PagedList<ClusterInner> list() {
```

Comment:
Is it any reason to change this return type?

Reply:
The conflict comes from `InnerSupportsListing`, which `ClustersInner` implements.